<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1409</width>
    <height>1084</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCL调参测试工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>171</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>应用栏</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>点云拼接</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>高度测量</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>平面度测量</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>体积计算</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ROI裁剪</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>异物检测</string>
     </property>
    </widget>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>781</width>
      <height>651</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="load_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>680</y>
      <width>791</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>编辑栏</string>
    </property>
    <widget class="QPushButton" name="LoadBUT">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_button">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>另存为</string>
     </property>
    </widget>
    <widget class="QPushButton" name="excuteButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>750</y>
      <width>931</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>可视化每一步的参数的</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>131</width>
      <height>731</height>
     </rect>
    </property>
    <property name="title">
     <string>测试脚本</string>
    </property>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>去除NAN点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>点云滤波</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>点云分割</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>特征筛选</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>特征匹配</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>加载点云</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 170, 127);
font: 20pt &quot;华文楷体&quot;;

</string>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>40</y>
      <width>240</width>
      <height>950</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>581</height>
       </rect>
      </property>
      <property name="title">
       <string>调参栏</string>
      </property>
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>551</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;这里面是每一个类型的参数调试&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>620</y>
        <width>201</width>
        <height>281</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;这里是编辑框，用来写备注的&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>功能栏</string>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>181</width>
       <height>621</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1409</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_X">
    <property name="title">
     <string>执行(X)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>可视化(V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>函数(P)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>算子(O)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>建议(S)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>助手（A）</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_X"/>
   <addaction name="menu_V"/>
   <addaction name="menu_P"/>
   <addaction name="menu_O"/>
   <addaction name="menu_S"/>
   <addaction name="menu_A"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/treelcon/res_treeItemIcon/folder.png</normaloff>:/treelcon/res_treeItemIcon/folder.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建工程</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/treelcon/res_treeItemIcon/Project.png</normaloff>:/treelcon/res_treeItemIcon/Project.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/treelcon/res_treeItemIcon/folder_original.png</normaloff>:/treelcon/res_treeItemIcon/folder_original.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
